ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml

```diff
@@ -8,13 +8,23 @@ ref:
     requests:
       cpu: 1000m
       memory: 2Gi
+  credentials:
+  - namespace: test-credentials
```

Comment:
Just a question for my own training ... is it true that all credentials reside in the `test-credentials` namespace? thx

Reply:
Most of them, yes - see https://github.com/openshift/release/blob/master/core-services/ci-secret-bootstrap/_config.yaml

Reply:
thanks